Bio/PDB/MMCIFParser.py

```diff
@@ -84,7 +84,7 @@ def _update_header_entry(self, target_key, keys, is_float=False):
             except (TypeError, IndexError):
                 continue
             if item != "?":
-                self.header[target_key] = item if not is_float else float(item)
+                self.header[target_key] = item
                 return
```

Comment:
I'd remove this `return` and replace it with `break`, since the function will anyway return `None` and the goal of the return is to break the for loop.

Reply:
done.